vercel.json

```diff
@@ -136,7 +136,7 @@
     },
     {
       "source": "/docs/start-building/create-project",
-      "destination": "https://www.ory.sh/docs/guides/console/create-project",
```

Comment:
Please instead create a new redirect for `/docs/guides/console/create-projec`

Reply:
I reverted this change, is probably better to address in a dedicated PR!